### PR TITLE
Removes Space Hotel

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Removes Space hotel and all its associated code.

One day, I'd like to revisit adding this, but in its current state, it's not particularly functional---it also utilizes some weird system to mimic multi-z, which is....less than ideal.

This is a whole slew of maps and templates that have to be maintained that have been non-functional for *years* now---there's not much sense keeping them around when they don't even work properly, to begin with.

We can look at porting TG's implementation of this, one day, but, for now, keeping it around is just a headache to maintain.

. . . the way must be prepped. . .

:cl: Fox McCloud
del: removes the defunct and unused Space Hotel away mission
/:cl:

### CODE REVIEW

File: code/game/turfs/unsimulated.dm [hunk 1]
```diff
@@ -12,34 +12,3 @@
 
 /turf/unsimulated/acid_act(acidpwr, acid_volume, acid_id)
 	return 0
-
-/turf/unsimulated/floor/plating/vox
-	icon_state = "plating"
-	name = "plating"
-	nitrogen = 100
-	oxygen = 0
```
Comment (AffectedArc07) [inline code/game/turfs/unsimulated.dm:20]: Intended for this PR?

File: code/game/turfs/unsimulated/floor.dm [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
 /turf/unsimulated/floor
 	name = "floor"
 	icon = 'icons/turf/floors.dmi'
 	icon_state = "Floor3"
 
+/turf/unsimulated/floor/plating/vox
+	icon_state = "plating"
+	name = "plating"
+	nitrogen = 100
+	oxygen = 0
```
Comment (AffectedArc07) [inline code/game/turfs/unsimulated/floor.dm:11]: Nevermind just saw this

File: code/game/turfs/unsimulated.dm [hunk 3]
```diff
@@ -12,34 +12,3 @@
...
-
-/turf/unsimulated/floor/plating/snow
-	name = "snow"
-	icon = 'icons/turf/snow.dmi'
-	icon_state = "snow"
-	temperature = T0C
-
-/turf/unsimulated/floor/plating/snow/concrete
```
Comment (AffectedArc07) [inline code/game/turfs/unsimulated.dm:28]: `snow/concrete`

Sorry **WHAT?**

Comment (FreeStylaLT) [reply code/game/turfs/unsimulated.dm:28]: image

**Review by AffectedArc07 — APPROVED** [review]
Purge it

**Comment by Carthusia** [comment]
You can check out anytime, but you can never leave.

Goodbye hotel california

**Review by SteelSlayer — APPROVED** [review]
I found two references to the `/turf/unsimulated/wall/other` that you're removing if you want to remove those as well. In `\code\game\machinery\overview.dm` there are two of these sections:
```dm
if("/turf/unsimulated/wall", "/turf/unsimulated/wall/other")
	colour  = rgb(140,140,140)
```

You can remove the `"/turf/unsimulated/wall/other"` from here.

Other than that, LGTM.

File: code/game/turfs/unsimulated.dm (hunk 3 above)
Comment (Fox-McCloud) [reply code/game/turfs/unsimulated.dm:28]: Look at the git history and everything will make sense.

Comment (AffectedArc07) [reply code/game/turfs/unsimulated.dm:28]: Name checks out

**Comment by DevL-Whats-a-jpg** [comment]
Welcome to the Hotel 2D spessman
Such a deadly place (Such a deadly place)
Such a deadly face

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
Get this out the codebase

**Pull request merged**